lib/enums.js

```diff
@@ -0,0 +1,11 @@
+const enums = exports;
```

Comment:
Maybe we can move enums into our utils folder? Eventually I was thinking a constants file can go in that folder as well.

Reply:
`utils` folder will grow to contain more files - utils for specific domains. 

I don't think it should be mixed with enums/constants, which should have they own folder eventually.